common/choices.py

```diff
@@ -1,3 +1,20 @@
+from django.db.models import TextChoices
+
+
+class IncidentCategoryChoices(TextChoices):
```

Comment:
Is there any way to query these dynamically or do they have to be hardcoded?

Reply:
Actually, now that you mention it, I think there is. Let me try to get that going.

Reply:
In fact, we're already using that technique on the tags field! :headslap: